Makefile

```diff
@@ -0,0 +1,53 @@
+PYTHON ?= $(shell command -v python3 2>/dev/null || command -v python)
+DESTDIR ?= /
+export PODMAN_VERSION ?= '1.80'
```

Comment:
It would be nice if we could manage to grab the version number from somewhere.  But that's past my Makefile skills.